db.php

```diff
@@ -33,20 +33,22 @@
 
     if (isset($_POST["submit"])) {
 
-        $product_varient = $_POST['product'];
-        $gift_varient = $_POST['gift'];
+    $fee = $_POST['fee'];
+
 
 
         $check = "SELECT * FROM fee where store_name = '$store' ";
         $rs = mysqli_query($con, $check);
         $data = mysqli_fetch_array($rs, MYSQLI_NUM);
-        if ($data[0] > 1) {
-        $query = "UPDATE INTO `fee`(`amount`) VALUES ('$product_varient') where store_name = '$store' ";
+        
```

Comment:
@shank-code coding standard not proper, you some use of classes etc like you saw in scraping project. Not much impressed with code pattern, it's like something done by fresher